Formula/gdu.rb

```diff
@@ -0,0 +1,18 @@
+class Gdu < Formula
+  desc "Disk usage analyzer with console interface written in Go"
+  homepage "https://github.com/dundee/gdu"
+  url "https://github.com/dundee/gdu/archive/v2.2.0.tar.gz"
+  sha256 "94f4faaee6c1676b73bb75f031cc3d004b8df713fb12d1dfc90ed592bf302df7"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags=\"-s -w -X 'main.AppVersion=v2.2.0'\"", "."
+    bin.install "gdu" => "gdu"
+  end
+
+  test do
+    system "#{bin}/gdu", "-v"
```

Comment:
Version tests rarely ever error out when they need to, so we'll need a more substantial test than this. Cf. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Ok, I will add more tests. Thanks!

Reply:
I added some basic test for content of help. All the other features unfortunately require user input.

Reply:
Can you write some text into a file and check the usage of that file or the directory that contains it? https://github.com/dundee/gdu#usage

If you need some other content to populate the test directory, you can use the text fixtures: https://github.com/Homebrew/brew/tree/master/Library/Homebrew/test/support/fixtures

Reply:
Could you also assert the output of gdu version? So we can be sure there is not breakage ever of version variable setting at build time.

Reply:
> Can you write some text into a file and check the usage of that file or the directory that contains it? https://github.com/dundee/gdu#usage
> 
> If you need some other content to populate the test directory, you can use the text fixtures: https://github.com/Homebrew/brew/tree/master/Library/Homebrew/test/support/fixtures

I cannot right now. Gdu has no text output, only GUI.

Reply:
> Could you also assert the output of gdu version? So we can be sure there is no breakage ever of version variable set at build time.

Assert added, thanks!

Reply:
Don't check the output then; we just need to check that it'll run without producing an error.